.github/workflows/test-rector.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0']
+        php-versions: ['8.0', '8.1']
```

Comment:
Same here: Rector doesn't really care about its native version; running in 7.4 was just a safety.

Reply:
Done.